Podfile.lock

```diff
@@ -1,12 +1,12 @@
 PODS:
-  - Mapbox-iOS-SDK (4.1.0)
+  - Mapbox-iOS-SDK (4.1.1)
```

Comment:
Should we be updating to 4.1.1 in this PR?

Reply:
Thanks! Looks like there is an open PR to address that (#193)